beacon_node/beacon_chain/Cargo.toml

```diff
@@ -15,10 +15,10 @@ serde_derive = "1.0"
 slog = { version = "^2.2.3" , features = ["max_level_trace"] }
 sloggers = { version = "^0.3" }
 slot_clock = { path = "../../eth2/utils/slot_clock" }
-eth2_ssz = { path = "../../eth2/utils/ssz" }
-eth2_ssz_derive = { path = "../../eth2/utils/ssz_derive" }
+eth2_ssz = "0.1"
```

Comment:
I guess this is good for dog-fooding our crates.io release, but during development it's often nice to just be tweaking the local version of the crate and having that used instantly throughout the rest of the project. With pinned versions you'd have to do a lot more work setting up Cargo overrides, etc.

Reply:
The problem is that cargo can't identify that the `eth2_ssz` specified by path is the same as the one from crates.io.

When we publish to crates.io we can't use paths, so we start to get this problem where some crates use crates.io, others use paths and the compiler can no longer guarantee they're all the same.

Reply:
Ohhhh, that makes a lot of sense. Lets stick with crates.io deps then and I'll just have some copypasta on-hand for overriding